README.rst

```diff
@@ -147,52 +151,71 @@ taking priority.
 
     class SampleQuotesSpider(scrapy.Spider):
         name = "sample_quotes"
+        start_urls = ["https://quotes.toscrape.com/"]
 
         custom_settings = {
-            "ZYTE_API_DEFAULT_PARAMS": {
-                "geolocation": "US",  # You can set any Geolocation region you want.
-            }
+            "ZYTE_API_TRANSPARENT_MODE": True,
         }
 
+        def parse(self, response):
+            print(response.text)
+            # "<html>…</html>"
+
+
+Sending requests with manually-defined parameters
+-------------------------------------------------
+
+To send a Scrapy request through Zyte API with manually-defined parameters,
+define your Zyte API parameters in the ``zyte_api`` key in
+`Request.meta <https://docs.scrapy.org/en/latest/topics/request-response.html#scrapy.http.Request.meta>`_
+as a ``dict``.
+
+The only exception is the ``url`` parameter, which should not be defined as a
+Zyte API parameter. The value from ``Request.url`` is used automatically.
+
+For example:
+
+.. code-block:: python
+
+    import scrapy
+
+
+    class SampleQuotesSpider(scrapy.Spider):
+        name = "sample_quotes"
+
         def start_requests(self):
             yield scrapy.Request(
-                url="http://quotes.toscrape.com/",
-                callback=self.parse,
+                url="https://quotes.toscrape.com/",
                 meta={
                     "zyte_api": {
-                        "browserHtml": True,
-                        "javascript": True,
-                        "echoData": {"some_value_I_could_track": 123},
+                        "httpResponseBody": True,
+                        "httpResponseHeaders": True,
```

Comment:
I think that using browserHtml might be better here, as it shows a real use case for using the manual mode.

Reply:
I ended up covering both, ``browserHtml`` as a “normal” example, and this as an example of the need to include headers for proper decoding. Not sure if having the latter as an example is overkill.